src/Workspaces/MSBuildTest/DotNetSdkMSBuildInstalled.cs

```diff
@@ -3,55 +3,44 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Linq;
 using Microsoft.Build.Locator;
 using Roslyn.Test.Utilities;
 
 namespace Microsoft.CodeAnalysis.MSBuild.UnitTests
 {
     internal partial class DotNetSdkMSBuildInstalled : ExecutionCondition
     {
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Curious: is there a reason you prefer this over `#if NETCOREAPP`? don't think we use earlier than `netcoreapp3.1` anymore.